test/unit/lifecycle/properties.js

```diff
@@ -25,7 +26,11 @@ describe('lifecycle/property', () => {
   });
 
   describe('props declared as attributes with ES2015 classes are linked', () => {
+    const skip = !classStaticsInheritance();
```

Comment:
Oh, right, not sure. I do know many browsers shipped `setPrototypeOf` before the class syntax, though.